dictionaries/companies/dict/companies.txt

```diff
@@ -2078,3 +2078,4 @@ ru
 synology
 undraw
 university
+CSAIL
```

Comment:
Please add it to `dictionaries/companies/src/companies.txt`

Reply:
in new pull request i fix it 
https://github.com/streetsidesoftware/cspell-dicts/pull/2687

Reply:
Thank you.